README.md

```diff
@@ -1,6 +1,6 @@
 # wordpress-transient
 
-In this little snippet of code we can see how to cache the get_posts() in wordpress,
+In this snippet of code we can see how to cache the **get_posts()** in wordpress,
```

Comment:
wouldn't it be nicer to replace little with tiny?

Reply:
you're right, thanks for feed back, I'll commit it right now